docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.rst

```diff
@@ -0,0 +1,108 @@
+Create a Debezium source connector for PostgreSQL
+=================================================
+
+The Debezium source connector extracts the changes committed to the transaction log in a relational database, such as PostgreSQL, and provides them in a standard format into an Apache Kafka topic where can be transformed and read by multiple consumers. 
+
+.. Tip::
+
+    Sourcing data from a database into Apache Kafka decouples the database from the set of consumers. Once the data is in Apache Kafka, multiple applications can access it without adding any additional query overhead to the source database.
+
+.. _connect_debezium_pg_source_prereq:
+
+Prerequisites
+-------------
+
+To setup a Debezium source connector pointing to PostgreSQL, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+Furthermore you need to collect the following information about the source PostgreSQL database upfront:
+
+* ``PG_HOST``: The database hostname
+* ``PG_PORT``: The database port
+* ``PG_USER``: The database user to connect
+* ``PG_PASSWORD``: The database password for the ``PG_USER``
+* ``PG_DATABASE_NAME``: The database name
+* ``SSL_MODE``: The `SSL mode <https://www.postgresql.org/docs/current/libpq-ssl.html>`_
+* ``PG_TABLES``: The list of database tables to be included in Apache Kafka; the list must be in the form of ``schema_name1.table_name1,schema_name2.table_name2``
+* ``PG_PUBLICATION_NAME``: The name of the `PostgreSQL logical replication publication <https://www.postgresql.org/docs/current/logical-replication-publication.html>`_, if left empty, ``debezium`` is used as default.
+* ``PG_SLOT_NAME``: name of the `PostgreSQL replication slot <https://developer.aiven.io/docs/products/postgresql/howto/setup-logical-replication>`_, if left empty, ``debezium`` is be used as default
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service, setting needed only when using Avro as data format
```

Comment:
I'd be tempted to change `setting needed only` to `only needed` in all the items where that text is used.
That is slightly shorter, and I think it is already clear that we are referring to *that* particular setting / piece of information.
(also, it's the first time the word "setting" is being used in this way)

Reply:
Good suggestion, thanks!